File: Python/tests/test_igt_orl_sigmainvgam.py
import pytest

from hbayesdm.models import igt_orl_sigmainvgam


def test_igt_orl_sigmainvgam():
    _ = igt_orl_sigmainvgam(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_igt_orl_sigmainvgam.R
context("Test igt_orl_sigmainvgam")
library(hBayesDM)

test_that("Test igt_orl_sigmainvgam", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(igt_orl_sigmainvgam(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
